src/test/resources/testDataFiles/LoginUser.json

```diff
@@ -10,5 +10,8 @@
     "AccountCreated": "ACCOUNT CREATED!",
     "AccountDeleted": "ACCOUNT DELETED!",
     "Login": "Login to your account"
-  }
+  },
+  "RegisteredUserUsername": "abdelrahman",
+  "RegisteredUserEmail": "abdelrahman@gmail.com",
```

Comment:
We agreed not to use our names as test data. Please change those as agreed to be something like "automation" or "bot", etc....